cmd/internal/switcher/templates/configdb.go

```diff
@@ -14,42 +14,36 @@ import (
 )
 
 const (
-	// Tried to use "metal-core" name for the file. It doesn't work.
+	// Tried to use name that includes "-" for the file. It doesn't work.
 	// Systemd transforms "-" to "\" when %I specifier is used.
-	metalCoreConfigdb     = "/etc/sonic/metal.json"
-	metalCoreConfigdbTmp  = "/etc/sonic/metal.tmp"
-	configdbReloadService = "write-to-db"
+	metalCoreVrfConfig    = "/etc/sonic/vrfs.json"
```

Comment:
we could create a directory with all files which are metal related, for example /etc/sonic/metal.

Reply:
Done

Reply:
The SONiC Role must be updated accordingly to create this directory